integrations/azure/ortpipeline.yml

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2020 Bosch.IO GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# Ort starter pipeline: Customize to your need
+
+trigger: none
+
+parameters:
+  - name: project_name
+    displayName: "Project name (for display purposes only)"
+    type: string
+  - name: project_vcs_url
+    displayName: "Project Version Control URL (e.g. github .git address)"
+    type: string
+  - name: project_vcs_revision
+    displayName: "Project Revision (prefix Git tags with 'refs/tags/'), master for last commit"
+    type: string
+    default: master
+  - name: use_credentials
+    displayName: "Use repository credentials as defined in variable group ort_repo_credentials"
+    type: boolean
+    default: false
+
+  - name: ort_project_vcs_url
+    displayName: "Config Version Control URL (e.g. github .git address, use 'none' if the config is already provided)"
+    type: string
+    default: 'none'
+  - name: ort_project_vcs_revision
+    displayName: "Config Revision (prefix Git tags with 'refs/tags/'), master for last commit"
+    type: string
+    default: master
+  - name: ort_use_credentials
+    displayName: "Use credentials for ort as defined in variable group ort_config_credentials (only necessary when different repository is used)"
+    type: boolean
+    default: false
+
+  - name: log_level
+    displayName: "Log level"
+    type: string
+    default: "default"
+    values: [ "default", "info", "debug" ]
+  - name: stacktrace
+    displayName: "Stacktrace"
+    type: boolean
+    default: false
+
+  - name: allow_dynamic_versions
+    type: boolean
+    displayName: "Allow dynamic versions of dependencies"
+    default: false
+  - name: use_clearly_defined
+    type: boolean
+    displayName: "Use package curation data from ClearlyDefined"
+    default: true
+
+name: ${{parameters.project_name}}_$(Date:yyyyMMdd)
+pool:
+  vmImage: 'Ubuntu-20.04'
+
+variables:
+  - name: DOCKER_BUILDKIT
+    value: 1
+  - ${{ if eq(parameters.use_credentials, True) }}:
+      - group: 'ort_repo_credentials'
+  - ${{ if eq(parameters.ort_use_credentials, True) }}:
+      - group: 'ort_config_credentials'
+
+stages:
+  - stage: run_ort_pipeline
+    displayName: "Run Ort Starter Pipeline"
+    jobs:
+      - job: run_all_steps
+        displayName: "Build and Run Ort"
```

Comment:
Change "Build and Run Ort" to "Build and run ORT"

Reply:
FYI, this wasn't addressed here, but I've addressed it as part of https://github.com/oss-review-toolkit/ort/pull/3541.